mappers/modbus-dmi/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: modbus-mapper
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: modbusmapper
+  template:
+    metadata:
+      labels:
+        app: modbusmapper
+    spec:
+      hostNetwork: true
+      containers:
+        - name: modbus-mapper-container
+          image: modbusmapper:v1.0
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - name: config-volume
```

Comment:
we need to mount the directory of unix domain sock to the mapper container

Reply:
mount `/var/run` directory right?